src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -59,6 +59,7 @@
   private static final String DEV_APPSERVER_PY = "bin/dev_appserver.py";
   private static final String JAVA_APPENGINE_SDK_PATH =
       "platform/google_appengine/google/appengine/tools/java/lib";
+  private static final String JAVA_COMPONENTS_PATH = "platform/gcd/.appengine";
```

Comment:
FWIW: although my installation is out of date, I did have `app-engine-java`, but I don't have anything in `platform/gcd`.